src/Domain/Event/EventBufferInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Domain\Event;
+
+use Broadway\Domain\DomainMessage;
+
+interface EventBufferInterface
+{
+    /**
+     * Add DomainMessage to buffer.
+     *
+     *
+     * @throws \App\Domain\Event\Exception\BufferOverflowException
+     */
+    public function add(DomainMessage $event);
+
+    /**
+     * Check if buffer is full.
+     *
+     * @return bool
+     */
+    public function isFull();
```

Comment:
You could specify the return type `: bool`. If you do the return block from phpdoc isn't needed.